__tests__/app.tests.js

```diff
@@ -34,11 +34,32 @@ describe("TOPICS TESTS", () => {
 });
 
 describe("ARTICLES TESTS", () => {
+  describe("GET /api/articles", () => {
+    test("Returns 200 status code and array of all articles sorted by date in descending order", async () => {
+      const res = await request(app).get("/api/articles").expect(200);
+      expect(res.body.articles).toBeInstanceOf(Array);
+      expect(res.body.articles.length).toBe(12);
+      expect(res.body.articles).toBeSortedBy("created_at", {
+        descending: true,
+      });
+      res.body.articles.forEach((article) => {
+        expect(article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
```

Comment:
shouldn't get body when we get all articles

Reply:
I need to read the task description more carefully. Removed now.